src/Sarif/Visitors/SarifVersionOneToCurrentVisitor.cs

```diff
@@ -1194,23 +1192,48 @@ internal StackFrame CreateStackFrame(StackFrameVersionOne v1StackFrame)
             return stackFrame;
         }
 
+        private const string DottedQuadFileVersionPattern = @"[0-9]+(\.[0-9]+){3}";
```

Comment:
>@"[0-9]+(\.[0-9]+){3}" [](start = 60, length = 22)

Are version numbers globalized? i.e. should commas delimiters be accepted?

Reply:
Not globalized, no commas. Think "IP address". Dots are part of the format.

---
In reply to: [314364505](https://github.com/microsoft/sarif-sdk/pull/1647#discussion_r314364505) [](ancestors = 314364505)